### PR TITLE
Add log test

### PR DESCRIPTION
**Pull request by xlgao-zju**
1. Remove useless code about e2e.
2. Remove runv in build hyperd, since we do not need it anymore.
3. Add test for log.

### CODE REVIEW

**Comment by xlgao-zju** [comment]
cc @feiskyer

File: test/e2e/utils.go [hunk 1]
```diff
@@ -221,3 +266,17 @@ func containerFound(containers []*runtimeapi.Container, containerID string) bool
 	}
 	return false
 }
+
+func checkLog(podConfig *runtimeapi.PodSandboxConfig, logPath string, expectedContext string) {
```
Comment (feiskyer) [inline test/e2e/utils.go]: Could we also verify log contents? the format of log file is same with docker's json-log.

Comment (xlgao-zju) [inline test/e2e/utils.go]: @feiskyer You mean I should import `github.com/docker/docker/daemon/logger`? Dose this pkg have the function which can verify log contents?

Comment (feiskyer) [inline test/e2e/utils.go]: `github.com/docker/docker/pkg/jsonlog`, refer https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/kuberuntime_logs.go#L246

**Comment by xlgao-zju** [comment]
@feiskyer updated.

**Comment by feiskyer** [comment]
LGTM. Thanks.

**Pull request merged**